integration-tests/Makefile

```
@@ -59,11 +61,18 @@ $(foreach element,$(ALL_TESTS),$(eval $(call make-test-target-dockerized,$(eleme
 
 .PHONY: build
 build:
-	mkdir -p bin
-	go test -tags bench,k8s -c -o bin/collector-tests
+	mkdir -p bin/$(GOARCH)
+	CGO_ENABLED=0 GOOS=linux GOARCH=$(GOARCH) go test -tags bench,k8s -c -o bin/$(GOARCH)/collector-tests
+
+.PHONY: build-all
+build-all:
+	GOARCH=amd64 make build
+	GOARCH=arm64 make build
+	GOARCH=s390x make build
+	GOARCH=ppc64le make build
```

Might be worth adjusting this so we can use `-j` to run them in parallel 
e.g.

```suggestion
ARCHS := amd64 arm64 s390x ppc64le

build-%:
  GOARCH=$(*) make build

build-all: $(addprefix build-,$(ARCHS))

.PHONY: build-all build-%
```